packages/atomic/cypress/integration/result-list/result-components-v1/result-text.cypress.ts

```diff
@@ -0,0 +1,187 @@
+import {HighlightKeyword} from '@coveo/headless';
```

Comment:
Most tests here are based on those defined in the removed file with the same name below.